package.json

```diff
@@ -35,7 +35,8 @@
     "test:ci": "yarn build:main && yarn test",
     "lint": "lerna run --stream lint",
     "postinstall": "husky install",
-    "db:chain": "node ./node_modules/.bin/typeorm"
+    "db:chain": "node ./node_modules/.bin/typeorm",
+    "update:valid:target": "node ./scripts/update-valid-target.js"
```

Comment:
What if naming it `update:valid-target` to keep `valid target` in the save level